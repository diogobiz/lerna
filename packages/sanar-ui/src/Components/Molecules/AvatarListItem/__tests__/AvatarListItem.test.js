
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESAvatarListItem from '../'

it('renders correctly', () => {
    const component = (
        <ESAvatarListItem>
            <h1>ESAvatarListItem</h1>
        </ESAvatarListItem>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
