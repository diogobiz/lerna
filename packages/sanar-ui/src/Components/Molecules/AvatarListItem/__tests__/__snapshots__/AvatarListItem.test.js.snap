// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="ant-list-item es-list-view-item"
>
  <div
    className="ant-row-flex es-row es-avatar-list-item"
    style={
      Object {
        "marginLeft": -8,
        "marginRight": -8,
      }
    }
  >
    <div
      className="ant-col es-col"
      style={
        Object {
          "paddingLeft": 8,
          "paddingRight": 8,
        }
      }
    >
      <span
        className="ant-avatar es-avatar-list-item--avatar ant-avatar-circle"
        style={Object {}}
      >
        <span
          className="ant-avatar-string"
          style={
            Object {
              "opacity": 0,
            }
          }
        />
      </span>
    </div>
    <div
      className="ant-col es-col es-avatar-list-item__content"
      style={
        Object {
          "flex": 1,
          "paddingLeft": 8,
          "paddingRight": 8,
        }
      }
    >
      <h1>
        ESAvatarListItem
      </h1>
    </div>
  </div>
</li>
`;
